sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuInsertIntoHadoopFsRelationCommand.scala

```diff
@@ -121,7 +122,7 @@ case class GpuInsertIntoHadoopFsRelationCommand(
       val pathExists = fs.exists(qualifiedOutputPath)
       (mode, pathExists) match {
         case (SaveMode.ErrorIfExists, true) =>
-          throw new AnalysisException(s"path $qualifiedOutputPath already exists.")
+          TrampolineUtil.throwAnalysisException(s"path $qualifiedOutputPath already exists.")
```

Comment:
I saw that and decided not to use it because it's adding `Set mode as "overwrite" to overwrite the existing path.` which we didn't have in our original message.

Reply:
But that extra message is applicable here, and we would want that.  The message was updated in Spark and we didn't update accordingly.